src/oge/reference_tables/epa_eia_crosswalk_manual.csv

```diff
@@ -190,4 +190,66 @@ plant_id_epa,emissions_unit_id_epa,plant_id_eia,generator_id,start_year,end_year
 65373,CT-3,65373,CTG-3,,,Matched based on generator ID
 65373,CT-4,65373,CTG-4,,,Matched based on generator ID
 65373,CT-5,65373,CTG-5,,,Matched based on generator ID
-65373,CT-6,65373,CTG-6,,,Matched based on generator ID
\ No newline at end of file
+65373,CT-6,65373,CTG-6,,,Matched based on generator ID
+1702,A,1702,3,,,Matched based on fuel type and fuel consumption
+1702,B,1702,4,,,Matched based on fuel type and fuel consumption
+2503,BLR114,2503,GT1,,,Only one generator at this plant

```

Comment:
I don't think we should match just based on the number of generators. This plant is a CHP plant, so these unmapped boilers are likely used to generate steam rather than power. According to the CEMS data, all of these units only generate steam, and not electricity

Reply:
Ok. What do you propose then? Because all CAMD units must be matched to a generator.